ext/drx/scatter_gather_aarch64.c

```diff
@@ -233,6 +233,60 @@ get_scatter_gather_info(instr_t *instr, OUT scatter_gather_info_t *sg_info)
     }
 }
 
+instr_t *
```

Comment:
Actually, one more comment: would it be possible to add a test too? Maybe a scatter-gather with a bad address?

Reply:
But that would fail -- do you mean a simple client that introduces its own fault inside a scatter-gather region?

Reply:
Even the app can recover though right? With a custom signal handler. suite/tests/client-interface/drx-scattergather-x86.c and suite/tests/client-interface/drreg-test.c do it.

Reply:
I'm guessing you meant something else besides "Maybe a scatter-gather with a bad address": a fault in a scatter or gather load/store will hit the assert as support is not implemented.  I thought we would want a test of what this PR adds, which is not failing for a fault that is not from a scatter/gather address.

Reply:
Ah yes. We don't want to hit the assert at L756. So a faulting instr adding by the client in the scatter-gather region is what we want.